test/controllers/ArrivalMovementControllerSpec.scala

```diff
@@ -71,12 +74,15 @@ class ArrivalMovementControllerSpec
 
   val mockClock = mock[Clock]
 
+  val configuration = ConfigFactory.load(ConfigFactory.parseFile(new File("conf/test-application.conf")))
+
   override lazy val app = GuiceApplicationBuilder()
     .overrides(
       bind[Metrics].toInstance(new TestMetrics),
       bind[AuthAction].to[FakeAuthAction],
       bind[ArrivalConnector].toInstance(mockArrivalConnector),
-      bind[Clock].toInstance(mockClock)
+      bind[Clock].toInstance(mockClock),
+      bind[Configuration].toInstance(Configuration(configuration))
     )
```

Comment:
Isn't there a `configure` override we can just use instead?

Plus, why would we need to change it here, if the config entry isn't going to be read in this controller?

Reply:
the phase4 controller tests use the config from application.conf by default - It's all good if we have configured the property as `enable-phase-5 = false` , but when we change this to `true`, then  these phase4 controller tests will fail as we return `GONE` instead of `ACCEPTED`

Not aware of `configure` where we can override the properties -then we no need to create a separate config file

Reply:
Oh, of course, in the P4 tests it hits the endpoint as if it was a web server, rather than directly. Okay, that makes sense.